test/common.js

```diff
@@ -0,0 +1,19 @@
+var yaml = require('js-yaml');
+
+module.exports.getDiag = function (body) {
+    var yamlStart = body.indexOf('  ---');
+    var yamlEnd = body.indexOf('  ...\n');
+    var diag = body.slice(yamlStart, yamlEnd).split('\n').map(function (line) {
+        return line.slice(2);
+    }).join('\n');
+
+    // The stack trace will vary depending on where the code is run, so just
+    // strip it out.
+    var withStack = yaml.safeLoad(diag);
+    delete withStack.stack;
+    return withStack;
+}
+
+module.exports.stripFullStack = function (output) {
+    return output.replace(/^\W+at.*:\d+:\d+.*$\W/gm, '');
```

Comment:
The problem is, stacks change. All the time. Even running the same test in node version 0.10.0 (which, guessing by the test matrix, you guys want to support) gives a different stack:

```
    +++ found                                                                  
    --- wanted                                                                 
         actual: |-                                                            
           {}                                                                  
         stack: |-                                                             
           Error: should be equivalent                                         
    +          at Test.assert (/Users/fmurphy/src/tape/lib/test.js:216:54)     
    +          at Test.bound [as _assert] (/Users/fmurphy/src/tape/lib/test.js:65:32)
    +          at Test.deepEqual.Test.deepEquals.Test.isEquivalent.Test.same (/Users/fmurphy/src/tape/lib/test.js:384:10)
    +          at Test.bound [as deepEqual] (/Users/fmurphy/src/tape/lib/test.js:65:32)
    +          at Test.<anonymous> (/Users/fmurphy/src/tape/test/undef.js:36:11)
    +          at Test.bound [as _cb] (/Users/fmurphy/src/tape/lib/test.js:65:32)
    +          at Test.run (/Users/fmurphy/src/tape/lib/test.js:84:10)         
    +          at Test.bound [as run] (/Users/fmurphy/src/tape/lib/test.js:65:32)
    +          at Object.next [as _onImmediate] (/Users/fmurphy/src/tape/lib/results.js:71:15)
    +          at processImmediate [as _immediateCallback] (timers.js:309:15)  
       ...  
```

Because, at some point, someone refactored the timers core library in node (and added the `runCallback` and `tryOnImmediate` frames). What normalization method would reduce both stacks to the same value? One has more frames than the other...